libvast/vast/format/json.hpp

```diff
@@ -115,6 +117,8 @@ class reader final : public multi_layout_reader {
   std::unique_ptr<detail::line_range> lines_;
   caf::optional<size_t> proto_field_;
   std::vector<size_t> port_fields_;
+  mutable size_t invalid_line_count_ = 0;
+  mutable size_t unknown_layout_count_ = 0;
```

Comment:
We usually use `num_invalid_lines_` for counter-like variables. (This should go in the style guide, actually.)

Reply:
Alright, will rename.